### PR TITLE
Jd 2023 01 fixtures

### PR DESCRIPTION
**Pull request by goapunk**
* add fixtures for every module type
* for module types with multiple phases, add one module with different active phase for each phase
* add script to dump fixtures (to make future changes to fixtures easier)

### CODE REVIEW

File: meinberlin/fixtures/votes.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "fields": {
+      "salt": "aUotLnFYnNsZwNia",
+      "module": 11
+    }
+  },
+  {
+    "model": "meinberlin_votes.votingtoken",
+    "pk": 1,
+    "fields": {
+      "token": "votingtoken1",
```
Comment (goapunk) [inline meinberlin/fixtures/votes.json:30]: do we document things like this somewhere?

**Comment by goapunk** [comment]
wip because #4866  needs to be merged first

File: meinberlin/fixtures/votes.json (hunk 1 above)
Comment (phillimorland) [reply meinberlin/fixtures/votes.json:30]: I'm not sure, @Rineee @fuzzylogic2000 ?

**Review by phillimorland — APPROVED** [review]
working nicely!

File: meinberlin/fixtures/votes.json (hunk 1 above)
Comment (Rineee) [reply meinberlin/fixtures/votes.json:30]: I think not yet ..

**Pull request merged**